test/infrastructure/docker/hack/verify-deps.sh

```diff
@@ -1,50 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
This was a second way to verify modules outside of the `make verify-modules` target